### PR TITLE
PLAT-143566: Added screenshot tests for VirtualList and VirtualGridList

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added screenshot tests for VirtualList and VirtualGridList components.

### Links
[//]: # (Related issues, references)
PLAT-143566

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/VirtualList.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import ri from '@enact/ui/resolution';
+
+import Item from '../../../../Item';
+import VirtualList from '../../../../VirtualList';
+
+const renderItem = () => ({index, ...rest}) => { // eslint-disable-line enact/display-name, enact/prop-types
+	const items = ['Item 000', 'Item 001', 'Item 002', 'Item 003', 'Item 004', 'Item 005', 'Item 006', 'Item 007', 'Item 008', 'Item 009'];
+	const itemStyle = {
+		borderBottom: ri.unit(6, 'rem') + ' solid #202328',
+		boxSizing: 'border-box',
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/VirtualList.js:10]: ItemBorder is not used in sandstone.

File: tests/screenshot/apps/components/VirtualList.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			dataSize={10}
+			itemSize={ri.scale( 144)}
+			itemRenderer={renderItem(ri.scale(144))}
+			spacing={ri.scale(60)}
+		/>
+	</div>,
+	<div>
+		<VirtualList
+			dataSize={10}
+			horizontalScrollbar="visible"
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/VirtualList.js:33]: In order to use the 'visible' option, the 'focusableScrollbar' option should be set to 'true' or 'byEnter'.

File: tests/screenshot/apps/components/VirtualGridList.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+		<VirtualGridList
+			dataSize={items.length}
+			itemSize={{minWidth: ri.scale(688), minHeight: ri.scale(570)}}
+			itemRenderer={renderItem}
+		/>
+	</div>,
+	<div>
+		<VirtualGridList
+			dataSize={items.length}
+			horizontalScrollbar="visible"
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/VirtualGridList.js:58]: In order to use the 'visible' option, the 'focusableScrollbar' option must be set to 'true' or 'byEnter'.

**Comment by dongsuwon** [comment]
In addition, it seems better to add RTL test.

**Pull request merged**